Base.rte/Scenes/Objects/Bunkers/BunkerSystems/Ladders/LadderBG.lua

```diff
@@ -0,0 +1,71 @@
+function Create(self)
+	self.skipFrames = 2;
+	self.checkDelay = math.ceil(self.skipFrames * TimerMan.DeltaTimeMS);
+	self.idleDelay = self.checkDelay * 5;
+	self.checkTimer = Timer();
+	self.checkTimer:SetSimTimeLimitMS(self.idleDelay);
+	self.checkCounter = 0;
+	self.idleCheckLimit = 50;
+	
+	self.checkPos = self.Pos;
+	
+	self.width = (ToMOSprite(self):GetSpriteWidth()/5) - 1;
+	self.height = (ToMOSprite(self):GetSpriteHeight()/3) - 1;
+	
+	self.gib = CreateTerrainObject("Destroyed Background Ladder"); 
+	self.gib.Pos = Vector(self.Pos.X, self.Pos.Y);
+end
+function Update(self)
+	if self.PinStrength ~= 0 then
+		if self.checkTimer:IsPastSimTimeLimit() then
+			self.checkTimer:Reset();
+			self.Vel, self.AngularVel = Vector(), 0;
+			local moCheck = SceneMan:GetMOIDPixel(self.checkPos.X, self.checkPos.Y);
+			local actor;
+			if moCheck ~= rte.NoMOID then
+				local mo = MovableMan:GetMOFromID(MovableMan:GetMOFromID(moCheck).RootID);
+				if mo and IsAHuman(mo) then
+					actor = ToAHuman(MovableMan:GetMOFromID(mo.RootID));
+					local controller = actor:GetController();
+					local limbs = actor.FGLeg or actor.BGLeg or actor.FGArm or actor.BGArm;
+					if limbs and actor.Status == Actor.STABLE and not controller:IsState(Controller.BODY_JUMP) then
+						local velFactor, aimAngle, climb = 1 + actor.Vel.Magnitude * 0.3, actor:GetAimAngle(false), false;
```

Comment:
I know it adds a few lines, but I usually find it more readable to not do multiple variables on one line, except maybe when they're all the same or very short. In this case, with them all being quite different and 2 of them being relatively complex, it kinda obfuscates things.